scripts/generate_wasm.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
If you don't use any `bash` specific functionality i would suggest to use `#!/bin/sh`

Reply:
done